src/nvim/CMakeLists.txt

```diff
@@ -526,11 +526,6 @@ endfunction()
 
 set(NO_SINGLE_CHECK_HEADERS
   os/win_defs.h
```

Comment:
This may be checked on Windows, AFAIR I was suggesting to add this to NO_SINGLE_CHECK_HEADERS depending on the platform condition.